### ISSUE TITLE
Ember.js Example?

### ISSUE THREAD

**Issue by wukongrita** [floatinghotpot/cordova-admob-pro#634]
Hi,

I'm using [ember-cli-cordova](https://github.com/poetic/ember-cli-cordova) to make Ember.js be available in mobile version (Android). Do you have an example in implementing this plugin with Ember.js ?

Thank you...

**Comment by floatinghotpot** [comment]
No, we did not test with the tool. If you can find the cordova folder in your project, you can try it yourself to add the plugin with cordova CLI.

**Comment by wukongrita** [comment]
I already add the plugin with the tools, but I still do not know how/where to put the code to show the ads. Any example on how doing this with Ember.js?
